runhouse/rns/rns_client.py

```diff
@@ -630,3 +637,31 @@ def contents(self, name_or_path, full_paths):
         return folder(name=name_or_path, path=folder_url).resources(
             full_paths=full_paths
         )
+
+    async def asave_status_metrics_to_den(
+        self, status: ResourceStatusData, cluster_uri: str, api_server_url: str
+    ):
+        from runhouse.resources.hardware.utils import ResourceServerStatus
+
+        resource_info = dict(status)
```

Comment:
small nit - i think we should use .dict() for pydantic models (https://docs.pydantic.dev/1.10/usage/exporting_models/)